package.json

```
@@ -6,7 +6,8 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "run-s lint lint:js lint:scss",
```

I think we'll have an infinite loop here, since it will try to run `lint`, which will try to run `lint`, which will try to run `lint`...

```suggestion
    "lint": "run-s lint:js lint:scss",
```